perception/autoware_traffic_light_map_based_detector/README.md

```diff
@@ -9,7 +9,7 @@ Calibration and vibration errors can be entered as parameters, and the size of t
 ![traffic_light_map_based_detector_result](./docs/traffic_light_map_based_detector_result.svg)
 
 If the node receives route information, it only looks at traffic lights on that route.
-If the node receives no route information, it looks at a radius of 200 meters and the angle between the traffic light and the camera is less than 40 degrees.
+If the node receives no route information, it looks at a radius of `max_detection_range` and the angle between the traffic light and the camera is less than `traffic_light_max_angle_range`.
```

Comment:
Can you write this better please?

Reply:
I rewrote it as: https://github.com/autowarefoundation/autoware.universe/pull/6724/commits/e529247b56d761aa95e1c08828fc975a03d2d57b#diff-4d2876c605a3e0eae9ff8ffe9aba6aff9045cc5ac297b7149ed74b5793c82053R12